jujugui/static/gui/src/app/components/inspector-change-version/_inspector-change-version.scss

```diff
@@ -0,0 +1,24 @@
+.inspector-change-version {
+    @extend %scroll-children;
+
+    &__current {
```

Comment:
From https://en.bem.info/method/naming-convention/#modifier-name:

"A modifier name is delimited by a single underscore (_)."

We might be following a different naming convention?